lib/game_manager.rb

```diff
@@ -0,0 +1,19 @@
+require_relative '../classes/game'
+
+class GameManager
+  def initialize
+    @games_list = []
+  end
+
+  attr_reader :games_list
+
+  def add_game(title, multiplayer, last_played_at, publish_date, archived)
+    game = Game.new(title, multiplayer, last_played_at, publish_date, archived)
+    @games_list.push(game)
+    game
+  end
+
+  def authors_list
+    authors
```

Comment:
This **authors** variable is not defined anywhere. Hence, the following error is thrown when this line executes.
![image](https://github.com/henokkhm/microverse-module-4-ruby-group-capstone/assets/110997325/0fb2c54e-2244-465e-82e9-079db366004c)

Instead of putting the Authors in the GameManager, you can create an **AuthorsManager** class that works in the same way.